src/msg/async/Stack.h

```diff
@@ -214,6 +217,15 @@ enum {
   l_msgr_last,
 };
 
+enum {
+  l_msgr_labeled_first,
```

Comment:
Can't start from `0` here:
```suggestion
  l_msgr_labeled_first = l_msgr_last + 1,
```